bootstrap/bootstrap.go

```diff
@@ -201,6 +216,62 @@ func (b *Bootstrapper) activateNodeDomain(am core.ArtifactManager, cb *testutil.
 	return nil
 }
 
+func (b *Bootstrapper) activateRootMember(am core.ArtifactManager, cb *testutil.ContractsBuilder) (*core.RecordRef, error) {
+	instanceData, err := serializeInstance(memberContract.New("RootMember", b.rootPubKey))
+	if err != nil {
+		return nil, errors.Wrap(err, "[ ActivateRootMember ]")
+	}
+
+	rootMemberRef, err := am.ActivateObject(
+		core.RecordRef{}, core.RandomRef(),
+		*cb.Classes[member],
+		*b.rootDomainRef,
+		instanceData,
+	)
+	if err != nil {
+		return nil, errors.Wrap(err, "[ ActivateRootMember ]")
+	}
+	if rootMemberRef == nil {
+		return nil, errors.Wrap(err, "[ ActivateActivateRootMember ] couldn't create root member")
+	}
+
+	updateData, err := serializeInstance(&rootdomain.RootDomain{Root: *rootMemberRef})
+	if err != nil {
+		return nil, errors.Wrap(err, "[ ActivateRootMember ]")
+	}
+	_, err = am.UpdateObject(
+		core.RecordRef{}, core.RandomRef(),
+		*b.rootDomainRef, updateData,
+	)
+	if err != nil {
+		return nil, errors.Wrap(err, "[ ActivateRootMember ]")
+	}
+
+	return rootMemberRef, nil
```

Comment:
We can make `rootMemberRef` an argument of `Bootstrapper`, so we don't need to pass it as result here and accept in `activateRootWallet` as param